packages/core/src/transforms/resize.ts

```diff
@@ -46,7 +54,9 @@ export const resize: TransformFactory<ResizeOptions> = (config, ctx) => {
             : undefined
     const aspect = width && height
         ? width / height
-        : parseAspect(config.aspect)
+        : config.aspect
```

Comment:
```suggestion
        : parseAspect(config.aspect || config.ar)
```
why not use this?

Reply:
Great catch on my missing `ResizeOptions` for `ar`

> why not use this?
Yeah, there's no good reason... I skipped right past that cleaner `||` option and went straight for a nested ternary.  Just pushed a change to clean that up